pkg/packagekit/updates.less

```diff
@@ -10,6 +10,19 @@ tr.listing-ct-item {
     td.version {
         max-width: 18ex;
         word-wrap: break-word;
```

Comment:
I thought `word-wrap` was supposed to be removed? With it there, it doesn't work right in Chrome & Epiphany for me, at least in my static version.